doc/glossary.rst

```diff
@@ -0,0 +1,165 @@
+
+Filters
+-------
+
+For a guide on filtering, specific to electrophysiological data, check out this
+`paper <https://doi.org/10.1016/j.jneumeth.2014.08.002>`_ from the journal of neuroscience methods.
+
+For a more in depth tutorial, in code, check out the
+`MNE Filtering Tutorial <https://martinos.org/mne/stable/auto_tutorials/plot_background_filtering.html>`_.
+
+.. glossary::
+
+    Impulse Response
+        The response of a fitler when presented with an impulse; a single, brief input.
+    FIR
+        A Finite Impulse Response filter, meaning the response to a single input is finite, settling to zero.
+    IIR
+        An Infinite Impulse Response filter, meaning the response to a single input continue infinitely.
+    passband
+        The range (band) of frequencies that can pass through a filter.
+    stopband
+        The range (band) of frequencies that are attenuated (stopped) by a filter.
+    passtype
+        The type of filter, defined in terms of what frequency bands or ranges it passes through, or filters out.
+
+        * bandpass: a filter whose passband is a specific frequency band, bound by a low and high frequency point.
+        * bandstop: a filter that passes through all frequencies except a band region that is attenuated.
+        * lowpass: a filter whose passband is all frequencies below a filter frequency (low frequencies pass through).
+        * highpass: a filter whose passband is all frequencies above a filter frequency (high frequencies pass through).
+    transition band
+        The range of frequencies that are in the transition region between the passband and the stopband.
+    frequency response
+        The response profile of a filter, specifying the gain and phase shift applied by the filter at each frequency.
+
+Rhythms & Bursts
+----------------
+
+.. glossary::
+
+    burst
+        Periodic activity that lasts for a finite time , as in a 'burst of oscillatory activity'.
+
+Time Frequency
+--------------
+
+We currently have two general approaches to time frequency analyses:
+
+* those based on the Hilbert transform
+
+  * There is a scholarpedia article on using the
+    `Hilbert Transform for Brain Waves <http://www.scholarpedia.org/article/Hilbert_transform_for_brain_waves>`_
+  * See also this
+    `deep dive into Hilbert methods <http://www.rdgao.com/roemerhasit_Hilbert_Transform/>`_
+    from VoytekLab member Richard Gao.
+* wavelet based approaches.
+
+.. glossary::
+
+    frequency
+        The number of occurences over a unit of time.
+    phase
+        The position, at a point in time, on a waveform cycle.
+    amplitude
+        The magnitude of a signal, as the peak-to-trough.
+    period
+        A single cycle of a rhythm, defined as the time between two consecutive troughs (or peaks).
+    hilbert transform
+        A mathematical transform that derives the analyic representation of a signal, where the
```

Comment:
A mathematical transform that computes the "analytic signal", a complex-valued representation of a time-series (signal) that can be used to find its analytic amplitude and phase.